dora/src/vm/classes.rs

```diff
@@ -92,6 +92,68 @@ pub struct Class {
 }
 
 impl Class {
+    #[allow(dead_code)]
+    pub fn new(
```

Comment:
Looks like we should already use that method in `globaldef.rs` already? Then it wouldn't be dead code and gets test coverage.

Reply:
I suspect this method was copy/pasted from there, right?

Reply:
done.